test/library/packages/OrderedMap/types/mapOfArray.skipif

```diff
@@ -0,0 +1,3 @@
+# This .future is sometimes passing for multilocale/no-local
+CHPL_COMM!=none
+COMPOPTS <= --no-local
```

Comment:
This is borrowed from Map. I don't know if it's necessary for orderedMap. But I supposed it's suitable for both map and orderedMap...

Reply:
I just noticed though you're not adding a .future file for this test.

Map has the .future because valgrind reports a problem with that code.  It had that .skipif because the problem sometimes doesn't show up in multilocale (which start_test sees as a future that's now passing, which is something that requires attention.  So the skipif keeps it from running in cases where it might pass so that it only runs where it will fail.)

If the test works without valgrind errors for `OrderedMap`, then there'd be no reason to have a skipif.

I don't get any errors running the `OrderedMap` version of this test, with or without valgrind, with or without gasnet.  So I think you don't need the skipif.


(BTW, the Map test's skipif has been through a few changes since you copied it, `test/library/standard/Map/mapOfArray.skipif`.)

Reply:
removed

Reply:
So marking the initial comment as resolved isn't what makes the thread resolved, and now I can't undo it.  :joy: 

Removing the skipif as you've done resolves this issue, but I still can't figure out how to get giithub to reflect that.